src/lib/Hydra/Controller/Root.pm

```diff
@@ -341,10 +341,20 @@ sub evals :Local Args(0) {
 
     my $evals = $c->model('DB::JobsetEvals');
 
+    my @failEvals = [$c->model('DB::Jobsets')
+                       ->search({ "me.enabled" => 1, "me.hidden" => 0,
```

Comment:
There are some indentation issues here.
Also, it may be better to do only one `search` call, as it is implemented in the `search` method below (while I'm really not sure about this:/).

Reply:
I'm not sure quite what you mean about indentation "issues".  There are multiple dereferences, and each reference is somewhat long, so this separates each dereference to a separate line. The code works just fine (I run it locally).  Was your concern about functionality or just appearance, and did you have a specific recommendation regarding appearance?

In my experience it is typical for ORMs to perform local optimization of the generated SQL statement, and multiple refinements like this are often recommended as a way to specialize a base query.  Additionally the SQL server performs optimization on the query, so I don't expect this will result in any performance problems, especially when used with postgres.

Reply:
Ok, forget my comment about indentation and thx for your answer!